urllib3/util/ssl_.py

```diff
@@ -8,6 +8,13 @@
 HAS_SNI = False
 create_default_context = None
 
+# Maps the length of a digest to a possible hash function producing this digest
+HASHFUNC_MAP = {
+    16: md5,
+    20: sha1,
+    32: sha256,
+}
+
```

Comment:
It wasn't pulled out as a module-constant is because it's not shared with any other code. The only reason it was defined in a named dict is for self-documenting purposes and avoiding unnamed magic numbers inline. :)

We can keep this change if you prefer it, I have no strong feelings either way.